core/sql/comexe/ComTdbSort.h

```diff
@@ -201,11 +201,15 @@ class ComTdbSort : public ComTdb
   Int32 sortPartialKeyLen_;                             // 56-59
 
   UInt32 minimalSortRecs_;                              // 60-63
-  Float32 sortMemEstInMbPerCpu_;                        // 64-67
+  Float32 sortMemEstInKBPerNode_;                        // 64-67
   Float32 bmoCitizenshipFactor_;                        // 68-71
   Int32  pMemoryContingencyMB_;                        // 72-75
   UInt16 sortGrowthPercent_;                            // 76-77
-  char fillersComTdbSort_[18];                          // 78-95
+  char filler_1[2];                                     // 78-79
+  Int32 topNThreshold_;                                 // 80-83
+  Float32 estMemoryUsage_;                             // 84-87
+  Float32 bmoQuotaRatio_;                              // 88-92
```

Comment:
Comment should be 88-91, and on the next line 92-95

Reply:
done